### PR TITLE
Feedback Link

### LINKED ISSUE #343: Add help button and link to form
Let's add a help button on the top right of the screen like we had in previous designs. When a player clicks on the button, a separate tab opens and they're directed to [this google form](https://forms.gle/NVcg9ukPvUBYjw1u7). This will help us keep tabs on where people are getting stuck.

This feature request was born out of feedback we received during the demo where sometimes people would get stuck in the middle of the chapter but didn't know where to submit feedback about it.

### PR DESCRIPTION
**Pull request by benalleng**
Closes #343

Also cleans up some NavBar item classes to be more consistent

### CODE REVIEW

**Comment by ecurrencyhodler** [comment]
One nit is that we should probably change "get stuck?" to "need help?"

I think it translates easier to other languages. Other than that LGTM!

File: components/Navbar/HelpLink.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+'use client'
+
+import Link from 'next/link'
+import { Tooltip } from 'ui'
+
+export default function UserButton() {
```
Comment (afterburn) [inline components/Navbar/HelpLink.tsx:6]: File is called HelpLink, but function is called UserButton

File: components/Navbar/HelpLink.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        offset={0}
+        content={
+          <div className="flex">
+            <span className="whitespace-nowrap">Get stuck?</span>
+          </div>
+        }
+      >
+        <Link
+          className="relative flex h-full w-12 cursor-pointer items-center justify-center border-r border-white/25 text-center text-2xl text-white transition duration-100 ease-in-out hover:bg-black/25 md:w-16"
+          target={'_blank'}
```
Comment (afterburn) [inline components/Navbar/HelpLink.tsx:21]: You don't need curly braces if all you are passing is a string

File: components/Navbar/HelpLink.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+export default function UserButton() {
+  return (
+    <div className="flex-l flex h-full items-stretch">
+      <Tooltip
+        id={`navbar-tab-tooltip-3`}
+        position="bottom"
+        offset={0}
+        content={
+          <div className="flex">
+            <span className="whitespace-nowrap">Get stuck?</span>
```
Comment (afterburn) [inline components/Navbar/HelpLink.tsx:15]: Missing translation

File: components/Navbar/UserButton.tsx [hunk 4]
```diff
@@ -50,8 +52,11 @@ export default function UserButton() {
   }
 
   return (
-    <div>
-      <div className="ml-4 h-10 w-10">{renderIconButton()}</div>
+    <div className="flex-l flex h-full items-stretch">
+      <span aria-describedby="navbar-tab-user">
+        <span>{renderIconButton()}</span>
```
Comment (afterburn) [inline components/Navbar/UserButton.tsx:57]: Lets try to stay away from these types of render functions, I feel they make it difficult to grasp the overall html structure at a glance for other contributors.

I would also strongly suggest to keep the markup separate from the component's logic. In this instance I would classify `toggleModal` and `hideModal` as component logic, whereas `renderIconButton` is markup so it should come after `toggleModal` and `hideModal`. Ideally we just inline it like this though:

```jsx
<span aria-describedby="navbar-tab-user">
  {!hasMounted && <div />}
  {hasMounted && isLoggedIn && (
    <button
      id="navbar-tab-user-icon"
      onClick={() => setOpenSignInModal(true)}
      aria-label="profile"
      className="text-grey-300 relative flex h-full w-12 cursor-pointer items-center justify-center md:w-16"
    >
      <Avatar avatar={user.avatar} size={30} />
    </button>
  )}
  {hasMounted && !isLoggedIn && (
    <button
      id="navbar-tab-user-login"
      onClick={() => toggleModal(true)}
      aria-label="profile"
      className="text-grey-300 relative flex h-full w-12 cursor-pointer items-center justify-center md:w-16"
    >
      <UserIcon />
    </button>
  )}
</span>
```

**Review by afterburn — CHANGES_REQUESTED** [review]
Thanks @benalleng, requested a couple minor changes.

**Review by ecurrencyhodler — APPROVED** [review]
LGTM

**Pull request merged**